data/ecosystems/b/bitcoin.toml

```diff
@@ -1811,18 +1817,6 @@ url = "https://github.com/ElementsProject/reserves"
 [[repo]]
 url = "https://github.com/ElementsProject/rust-elements"
 
-[[repo]]
-url = "https://github.com/ElementsProject/rust-secp256k1-zkp"
```

Comment:
even tho these are forks of `BlockstreamResearch` repos, you can keep them because we have an algorithm for fork detection and tagging to deduplicate commit counts

Reply:
These aren't forks of BlockstreamResearch repos, they were moved FROM the ElementsProject repository TO the BlockstreamResearch repository.